plugins/modules/network_settings_workflow_manager.py

```diff
@@ -224,59 +224,86 @@
           site_name:
             description: >
               The name of the site provided as a path parameter, used
-              to specify where the IP sub-pool will be reserved.
+              to specify where the IP sub-pool will be reserved. (eg Global/Chennai/Trill)
             type: str
           settings:
             description: Network management details settings.
             type: dict
             suboptions:
               network_aaa:
-                description: Network V2's network_aaa.
+                description: Manages AAA (Authentication Authorization Accounting) for network devices.
                 suboptions:
+                  servers:
+                    description: Server type for managing AAA for network devices.
+                    choices: [AAA, ISE]
+                    default: ISE
+                    type: str
                   ip_address:
-                    description: IP address for AAA and ISE server (eg 1.1.1.1).
+                    description:
+                    - Primary IP address for the ISE server.
+                    - Secondary IP address for the AAA server.
```

Comment:
Can't we make ip_address to be the primary IP for ISE and AAA and network to be the PAN IP for ISE and Secondary IP for AAA?